priv/posts/marketing/20210714233950_getting-to-know-paulo-oliveira.md

```diff
@@ -28,7 +28,7 @@ Paulo F. Oliveira is a Portuguese software developer and a member of the
 Erlang Ecosystem Foundation. With a career spanning over 17 years, he
 has worked on the most diverse projects: from mobile games backends
 and internet portals, to NATO port surveillance systems and helicopter
-engine software, to space projects. And today, we get to know him.
+engine software, to space projects. And today, we got know him.
```

Comment:
This fix doesn't seem right. You mean "got to"?

Reply:
Fine catch 👍